robotiq_force_torque_sensor/urdf/robotiq_fts150.urdf.xacro

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
```

Comment:
Are there any best practices for these kind of xacros? Link and joint names do not appear to be harmonised / follow some kind of convention.